src/Stratis.Bitcoin.Features.PoA/PoABlockHeaderValidator.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using NBitcoin.Crypto;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockHeaderValidator
+    {
+        private readonly ILogger logger;
+
+        public PoABlockHeaderValidator(ILoggerFactory factory)
+        {
+            this.logger = factory.CreateLogger(this.GetType().FullName);
+        }
+
+        public void Sign(Key key, PoABlockHeader header)
+        {
+            ECDSASignature signature = key.Sign(header.GetHash());
```

Comment:
Do we need to make sure the signature is canonical before creating the BlockSignature?
```
ECDSASignature signature = key.Sign(header.GetHash());
signature = signatuee.MakeCanonical();
```

Reply:
It's always canonical be default.